index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Cual es tu nombre?')
+alert(`Bienvenido ${nombreUsuario}`)
```

Comment:
aca estaria bueno no usar interpolacion de strings
```suggestion
alert('Bienvenido ' + nombreUsuario)
```